### ISSUE TITLE
Playing on multiple servers

### ISSUE THREAD

**Issue by csarmi** [HearthSim/arcane_tracker#67]
Hello,

So I'm playing on all three servers. I switch between them quite often. Its causing me all sorts of problems in the app. Especially with the arena deck.
The question is whether its possible to use current server as a parameter when trying to guess which deck to use. Especially which arena deck to use.
Also, would it be possible to make more than one arena deck at the same time.

**Comment by martinbonnin** [comment]
Indeed, the app is quite hardcoded to support only one arena deck at a time at the moment. Not sure I can get the server information from the Log files. I'll look into it. Do you know how the Desktop trackers handle this by any chance ?

**Comment by csarmi** [comment]
no, sorry

**Comment by csarmi** [comment]
Anyway.
Just being able to clear a deck i.e. remove all cards would help too. Or
being able to delete arena deck.

**Comment by martinbonnin** [comment]
As of now, you can edit the deck and remove cards 1 by 1. Not ideal I know but that could be a workaround until we find a better solution.
